pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[project]
+name = "django-superform4"
```

Comment:
Do you explicitly want to change the project name?

I am asking since this is only about adding support for recent Python and Django versions and not about an architectural change in the project.